src/pages/Login/Login.js

```diff
@@ -0,0 +1,175 @@
+import React, { useEffect } from 'react';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+import styled from 'styled-components';
+import qs from 'qs';
+import {
+  restApiKey,
+  redirectUrl,
+  clientSecret,
+  kakaoAuthUrl,
+} from '../../config';
+
+const Login = () => {
+  const [searchParams] = useSearchParams();
+  const code = searchParams.get('code');
+  const navigate = useNavigate();
+
+  const body = qs.stringify({
+    grant_type: 'authorization_code',
+    client_id: restApiKey,
+    redirect_uri: redirectUrl,
+    code: code,
+    client_secret: clientSecret,
+  });
+
+  const getToken = async () => {
+    try {
+      await fetch('https://kauth.kakao.com/oauth/token', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded;charset=utf-8',
+        },
+        body,
+      })
+        .then(res => res.json())
+        .then(data => {
+          localStorage.setItem('kakao_token', data.access_token);
+          sendToken();
+        });
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  const sendToken = async () => {
+    try {
+      await fetch('http://10.58.5.73:8000/users/login', {
+        method: 'GET',
+        headers: { Authorization: localStorage.getItem('kakao_token') },
+      })
+        .then(res => res.json())
+        .then(data => {
+          localStorage.setItem('access_token', data.access_token);
+          navigate('/');
+        });
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  useEffect(() => {
+    if (code) {
+      getToken();
+    }
+  }, [code]);
```

Comment:
```suggestion
  useEffect(() => {
    code && getToken();
  }, [code]);
```
취향에 따라 다르겠지만 이렇게 줄이는 것도 한 번 참고해보세요!

Reply:
> 취향에 따라 다르겠지만 이렇게 줄이는 것도 한 번 참고해보세요!

@we-dohyeon 감사합니다! 도현님. 수정하겠습니다.